app/app/vhosts/new/route.js

```diff
@@ -39,6 +40,8 @@ export default Ember.Route.extend({
         return op.save();
       }).then( () => {
         let message = `Domain ${vhost.get('virtualDomain')} created`;
+        app.get('vhosts').unshiftObject(vhost);
```

Comment:
I would think that due to the fact that the `vhost` is created with an app (and app `hasMany('vhost')` and vhost `belongsTo('app')`) that the vhost would automatically be present in the app's `vhosts` property after this vhost is created. I would think that unshifting the vhost directly could result it in being added two times to the app.vhosts property.

If there's a specific ordering of vhosts that you want to ensure, using a CP on the vhost/index controller (creating a property like `sortedVhosts` or similar) may be worthwhile.